node-repository/src/main/java/com/yahoo/vespa/hosted/provision/provisioning/NodeResourceLimits.java

```diff
@@ -35,6 +38,17 @@ public void ensureWithinAdvertisedLimits(String type, NodeResources requested, A
             illegal(type, "diskGb", "Gb", cluster, requested.diskGb(), minAdvertisedDiskGb(requested, cluster.isExclusive()));
     }
 
+    // TODO: Move this check into the above when we are ready to fail, not just warn on this. */
```

Comment:
Consider mentioning "Vespa 9" in this comment so that we don't forget it.

Reply:
I think we can do it before ...